src/index.d.ts

```diff
@@ -7,6 +7,18 @@
 
 declare module 'material-ui-search-bar' {
   export interface SearchBarProps {
+      /**
+       * Whether to clear search on escape.
+       */
+      cancelOnEscape?: boolean;
+      /**
+       * Override or extend the styles applied to the component.
+       */
+      classes?: any;
```

Comment:
This should be `object`, or, more precisely match [the styles object](https://github.com/TeamWertarbyte/material-ui-search-bar/blob/master/src/components/SearchBar/SearchBar.js#L12):

```ts
{
  root?: object,
  iconButton?: object,
  iconButtonHidden?: object,
  iconButtonDisabled?: object,
  searchIconButton?: object,
  icon?: object,
  input?: object,
  searchContainer?: object
}
```
`

Reply:
I think declaring root as an object would 'break' compatibility with material-ui. The way I use classes is to [override them in my sass file](https://material-ui.com/customization/overrides/#overriding-with-classes). 

This might be a different issue, but if I'm not mistaken, the component shouldn't use the classes prop to override the default styles. In material-ui, that prop is used to declare classes that can be referred to in your CSS styles. But I should admit I'm a beginner with material-ui, so I might be wrong.

Or maybe just the name 'root' is what should be changed.

Reply:
Oh yeah. Everything should be a `string`, not an object. My bad. :sweat:

Reply:
Just to be clear, it should be:

```
{
  root?: string,
  iconButton?: string,
  iconButtonHidden?: string,
  iconButtonDisabled?: string,
  searchIconButton?: string,
  icon?: string,
  input?: string,
  searchContainer?: string
}
```

Reply:
Exactly, because every value is a class name.